go-controller/pkg/node/node.go

```diff
@@ -285,7 +286,7 @@ func (n *OvnNode) WatchEndpoints() {
 		UpdateFunc: func(old, new interface{}) {
 			epNew := new.(*kapi.Endpoints)
 			epOld := old.(*kapi.Endpoints)
-			if reflect.DeepEqual(epNew.Subsets, epOld.Subsets) {
+			if apiequality.Semantic.DeepEqual(epNew.Subsets, epOld.Subsets) {
```

Comment:
same

Reply:
this actually has a bug for not using apiequality

Reply:
https://github.com/kubernetes/kubernetes/pull/50583